isaric/parsers/predicovid.toml

```diff
@@ -154,18 +154,14 @@
   [visit.start_date]
     field = "symptom_start_date"
 
-  [visit.outcome]
+  [visit.outcome] # My concern is that the 'discharged' option will be overwritten by later empty rows given as "unknown"
```

Comment:
makes sense @pipliggins .. it could be that the data was not written/filled in, and not that the outcome is unknown.